client/js/lounge.js

```diff
@@ -764,10 +764,14 @@ $(function() {
 			.find(".chat")
 			.unsticky();
 
-		lastActive
+		var lastActiveChan = lastActive
 			.find(".chan.active")
 			.removeClass("active");
 
+		lastActiveChan
+			.find(".unread-marker")
+			.appendTo(lastActiveChan.find(".messages"));
```

Comment:
No, because the active *window* contains all the channels, so we need to find the specific *channel*. That was my initial solution before I ended up with a million unread markers in ever channels.